packages/grid/x-grid/src/tests/sorting.XGrid.test.tsx

```diff
@@ -29,7 +29,7 @@ describe('<XGrid /> - Sorting', () => {
 
   let apiRef: ApiRef;
 
-  const TestCase = (props: { rows?: any[]; model: SortModel }) => {
+  const TestCase = (props: { rows?: any[]; sortModel: SortModel }) => {
```

Comment:
renaming variable creates a mental overhead, I removed the shorthand for simplicity.